src/DocumentFormat.jl

```diff
@@ -18,7 +18,8 @@ struct FormatOptions
     comments::Bool
     docs::Bool
     lineends::Bool
-    kw::Bool
+    keywords::Bool
```

Comment:
How do the `keywords` and `kwarg` argument interact? My understanding from https://github.com/julia-vscode/julia-vscode/blob/master/package.json#L574 is that the `keywords` option is intended to do what `kwarg="none"` now does. Is there a reason to keep both options?

Reply:
They were mislabeled which is why they weren't doing what people expected. https://github.com/julia-vscode/julia-vscode/pull/1320 updates the descriptions